docs/guide/debugging.md

```diff
@@ -4,6 +4,18 @@ title: Debugging | Guide
 
 # Debugging
 
+## Terminal
+
+To debug a test file in your shell, add a `debugger` statement anywhere in your code, and then run `ndb`:
+
+```sh
+# install ndb globally
+npm install -g ndb
```

Comment:
```suggestion
npm install -g ndb

# alternatively, with yarn
yarn global add ndb
```

Reply:
Maybe we should mentioned `pnpm` too? I'm never sure if it's better to include all three, or do folks know?